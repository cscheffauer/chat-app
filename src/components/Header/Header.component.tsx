import React from 'react';
import './Header.scss';
interface Props {}

const Header = (props: Props) => {
	return (
		<div className='header'>
			<p>Status Meeting Standup</p>
		</div>
	);
};

export default Header;
