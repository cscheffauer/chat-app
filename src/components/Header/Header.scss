.header {
	height: 8vh; /* Fallback for browsers that do not support Custom Properties */
	height: calc(var(--vh, 1vh) * 8);
	width: 100%;
	display: flex;
	align-items: center;
	justify-content: center;
	p {
		margin-block-start: 0;
		margin-block-end: 0;
	}
}
