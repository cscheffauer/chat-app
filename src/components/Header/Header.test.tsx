import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header.component';


describe('Component / Header', () => {
    const HeaderWrapper = shallow(<Header />);

    it('expect to render Header component', () => {
        expect(HeaderWrapper).toMatchSnapshot();
    })

})


