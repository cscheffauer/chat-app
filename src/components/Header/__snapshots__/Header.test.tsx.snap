// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component / Header expect to render Header component 1`] = `
<div
  className="header"
>
  <p>
    Status Meeting Standup
  </p>
</div>
`;
